jobs/registry-change.js

```diff
@@ -157,10 +157,7 @@ module.exports = async function (
   // get config
   const keysToFindMonorepoDocs = _.compact(_.map(withMultiplePackageJSON, (group) => group[0].value.fullName.toLowerCase()))
   if (keysToFindMonorepoDocs.length) {
-    const monorepoDocs = (await repositories.query('by_full_name', {
```

Comment:
just one note: here we do `repositories.query()` and further down, you‘re replacing it with a call to `getAllDocs()` which does `repositories.allDocs()`, which is not the same :)

Reply:
I renamed it to 'queryDocs' for less confusion :)

Reply:
thanks!